examples/tick-ami/tick.json

```diff
@@ -123,6 +123,15 @@
   },{
     "type": "shell",
     "inline": [
+       "sudo apt-get install -y jq dos2unix",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-telegraf/install-telegraf",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-influxdb/install-influxdb",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-chronograf/install-chronograf",
+      "dos2unix /tmp/terraform-aws-influx/modules/install-kapacitor/install-kapacitor",
+      "sudo chmod +x /tmp/terraform-aws-influx/modules/install-telegraf/install-telegraf",
+      "sudo chmod +x /tmp/terraform-aws-influx/modules/install-influxdb/install-influxdb",
+      "sudo chmod +x /tmp/terraform-aws-influx/modules/install-chronograf/install-chronograf",
+      "sudo chmod +x /tmp/terraform-aws-influx/modules/install-kapacitor/install-kapacitor",
```

Comment:
Before modifying the code, I'd like to understand better why the permissions aren't working when you `clone` the repo on Windows. Is this a general Windows problem, where Git doesn't maintain file permissions? Or is this something specific to your particular Windows setup? Are you using `cmd`, Cygwin, something else?

Reply:
It seems to be a general Windows problem. So the repository was uploaded by a Linux machine (presumably) and thus the permissions were preserved, but as soon as they are downloaded by a Windows machine, Windows has no notion of permissions (it uses the file extension to decide if something is executable), so the information is thrown away. Then, when the files are re-uploaded to a Unix OS, they are all given some generic default permissions, and the special "executable" status is lost. This just re-adds the executable status if it happened to be lost, and doesn't hurt anything if uploaded from a Unix machine.